trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -0,0 +1,24 @@
+const { param } = require('express-validator');
+const { escape } = require('validator');
+
+const userParamEscapingSanitization = param('user').isString('User ID must be a string').escape();
```

Comment:
As the get user endpoint can be done via username and the username doesn't have any validation added to it when it's created I've had to opt to just sanitize and escape the input rather than anything else